nucypher/policy/models.py

```diff
@@ -596,3 +600,46 @@ def by_capsule(self, capsule):
                 if saved_capsule == capsule:
                     ursulas_by_capsules[ursula] = work_order
         return ursulas_by_capsules
+
+
+class Revocation:
+    revocation_splitter = BytestringSplitter((bytes, 7), (bytes, 32), Signature)
+
+    def __init__(self, arrangement_id: bytes,
+                       signer: 'SignatureStamp' = None,
+                       signature: Signature = None):
+        self.prefix = b'REVOKE-'
```

Comment:
> so that this specific string that gets signed is only for revoking 

Is this enforced somewhere?

> a plain signed `arrangement_id` may be of more use to a potential attacker somewhere

In what way?

Reply:
Yeah, I can't quickly think of a reason that it's useful.

But also: doesn't it make sense to have it be different than the `arrangement_id`?

Reply:
Not really, imho. What else would you use to identify it? Could be something policy specific and not arrangement specific, but that's what I see `Revocations` being -- arrangement specific.